string serial_pot_state
float32 pot